Python/initconfig.c

```diff
@@ -141,6 +151,16 @@ static const char usage_6[] =
 "   and end column offset) to every instruction in code objects. This is useful \n"
 "   when smaller cothe de objects and pyc files are desired as well as suppressing the \n"
 "   extra visual location indicators when the interpreter displays tracebacks.\n";
```

Comment:
```suggestion
"   extra visual location indicators when the interpreter displays tracebacks.";
```
(since other strings affected by  b9a0b19e3 *fputs→puts* transition have their `\n`s stripped)

Reply:
it’s only strings that are last of their section, newlines are still needed between items